.gitignore

```
@@ -1,6 +1,7 @@
+
 output/
-# config files are stored in projects/.../configs/icon/ directories. we save them in vitruvius
-**/icon/
+# config files are saved in vitruvius repo
+projects/**/configs/*.py
```

Are these the right ones?

```suggestion
packages/machine_learning/coco_autolabeler/configs/*.py
```